src/lib/Behat/Component/ContentTypePicker.php

```diff
@@ -53,13 +53,19 @@ public function verifyIsLoaded(): void
         $this->getHTMLPage()->find($this->getLocator('filterInput'))->clear();
     }
 
+    public function confirm(): void
```

Comment:
Interesting that these two views differ, maybe it's something worth talking about with the UX team? It's a minor thing, I know 😄